app/src/main/java/org/simple/clinic/editpatient/EditPatientModel.kt

```diff
@@ -71,6 +71,9 @@ data class EditPatientModel(
   val hasInputFields
     get() = inputFields != null
 
+  val isUserCountryIndiaAndAddingNHIDsFeatureEnabled
```

Comment:
We can simplify this variable name to say `canAddNHID`, WDYT?

Reply:
Also, do you think we can just move the `alternateId == null` check to this variable as well? 🤔